hack/check-marker.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+set -e
+
+source ./hack/lib/common.sh
+
+header_text "Running markdown link checker against changed markdown files"
+
+COMMIT_RANGE="$TRAVIS_COMMIT_RANGE"
+if [ -z "$COMMIT_RANGE" ] || ! git rev-list --quiet $COMMIT_RANGE; then
+	# Assume comparing current branch to master if we don't have $TRAVIS_COMMIT_RANGE
+	COMMIT_RANGE="master"
+fi
+
+find . -name '*.md' > .marker.tmp.all
+git diff --name-only $COMMIT_RANGE | grep '\.md$' | sed -e 's/^/.\//' > .marker.tmp.changed
+excludes=$(grep -vxF -f .marker.tmp.changed .marker.tmp.all | sed -e 's/^/--exclude /')
+rm .marker.tmp.*
+
+./hack/ci/marker --exclude website $excludes
```

Comment:
Could we ignore the README as well in order to solve this issue?

Reply:
Do we have any PRs that are blocked on README changes? I think those changes are pretty rare, so my vote would be to NOT exclude README.md for now (knowing that those PRs will still be blocked and we'll have to address them as they come up).